### PR TITLE
Isolate the file metedata generation script into its own DAG to align with others

### PR DESCRIPTION
**Pull request by frankhereford**
## Associated issues

This Issue closes [#14899](https://github.com/cityofaustin/atd-data-tech/issues/14899)

## Background

There's a bit of a backstory here, so please feel free to read all about it in [this slack thread](https://austininnovation.slack.com/archives/CM9MK950S/p1702038323069209).

### Background TLDR

* An ETL existed with a name that made us thing it was about tracking down missing PDFs, but it had a second thing it did which was populate crash records' `cr3_file_metadata`. 
* We disabled it as we were cleaning house in our migration from Airflow V1 to Airflow V2.
* In the course of trying to figure out how best to handle it, I pulled that secondary functionality out into its own docker image, which is contained in this PR.
  * With that functionality isolated, I removed the old, multi-purpose ETL. That deletion is reflected in this PR as well.

## Testing

**Steps to test:**

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: .github/workflows/build_docker_images.yml [hunk 1]
```diff
@@ -97,11 +97,11 @@ jobs:
...
+      - name: Build and push CR3 PDF metadata image
+        if: steps.changes.outputs.populate_cr3_file_metadata == 'true'
         uses: docker/build-push-action@v4
         with:
           platforms: linux/amd64,linux/arm64
-          context: atd-etl/process_missing_cr3_pdfs
+          context: atd-etl/populate_cr3_file_metadata
           push: true
-          tags: atddocker/vz-process-missing-cr3-pdfs:${{ github.ref == 'refs/heads/master' && 'production' || 'latest' }}
+          tags: atddocker/vz-pdfs:${{ github.ref == 'refs/heads/master' && 'production' || 'latest' }}
```
Comment (chiaberry) [inline .github/workflows/build_docker_images.yml:107]: I guess these are the only pdfs in vz so its not confusing, but what do you thinking about naming the image vz-cr3-metadata-pdfs or something similar?

Comment (frankhereford) [reply .github/workflows/build_docker_images.yml:107]: Yea, I think that's a really great idea @chiaberry, I'm going to push up a change with that in it.

Comment (frankhereford) [reply .github/workflows/build_docker_images.yml]: Thanks again for the suggestion, here's a commit naming it `vz-cr3-metadata-pdfs`: 1d59ca78a4de338761dd9f843092d832917467bc

File: atd-etl/populate_cr3_file_metadata/populate_cr3_file_metadata.py
Comment (mddilley) [inline atd-etl/populate_cr3_file_metadata/populate_cr3_file_metadata.py:1]: Just a side thought, I'm wondering how we can capture that this ETL is related to our ability to extract the CR3 narratives and diagrams. I might be thinking too hard about this, but this ETL feels like a dependency of the [cr3_extract_diagram](https://github.com/cityofaustin/atd-vz-data/tree/master/atd-etl/cr3_extract_diagram) ETL. Maybe there is a future version where we combine them into one DAG?

**Review by mddilley — APPROVED** [review]
I read through the code, and I think this does a great job of restoring the metadata population needed for the CR3 extract ETL. I dropped one thought about how this fits in with that ETL for discussion, but this is 🚢 to me. 🙏

**Review by patrickm02L — APPROVED** [review]
This approach is good to go! 🥏

**Comment by frankhereford** [comment]
Thanks y'all. I'm going to merge this when I get to my desk after lunch, and I'll toss up an Airflow PR to get this automated. Thanks again.

**Pull request merged**